src/Database/Log/LoggingStatement.php

```diff
@@ -109,15 +109,37 @@ public function bindValue($column, $value, $type = 'string')
      * Sets the logger object instance. When called with no arguments
      * it returns the currently setup logger instance
      *
+     * @deprecated 3.5.0 Use getLogger() and setLogger() instead.
```

Comment:
I agree, we should have a) section about the order of the doc block annotations in our [coding standards page in the CookBook](https://book.cakephp.org/3.0/en/contributing/cakephp-coding-conventions.html#commenting-code) and b) a sniff that can bug people, so I don't need to :trollface: 

@dereuromark I just searched for ``@deprecated`` in the 3.x code and my findings (samples) disagree with your order.

It seems the following is used most often (which I also remember we used normally):

````
@param
@return
@throws
@other
````

[Search on GitHub repo for "deprecated"](https://github.com/cakephp/cakephp/search?p=3&q=deprecated&type=&utf8=%E2%9C%93).

But we _could_ use the yet to be written sniff to tell us which order results in more violations in the current code and we then use the one that results in the least violations.
Though, I still prefer the ordering above, as it makes more sense to me (order of importance & of usage).